### PR TITLE
Facebook and Google login

### PR DESCRIPTION
**Pull request by khanhlyvu30**
TAG : https://edu-redmine.sun-asterisk.vn/issues/28957?issue_count=28&issue_position=1&next_issue_id=28507
WHAT can login with Facebook and Google

Screenshot_30
Screenshot_29
Screenshot_28

### CODE REVIEW

**Review by TranVanPine — APPROVED** [review]

**Review by NguyenDungB — APPROVED** [review]

**Comment by khanhlyvu30** [comment]
ready

File: imusic/settings.py [hunk 1]
```diff
@@ -133,5 +141,43 @@
...
+            'gender',
+            'updated_time'],
+        'EXCHANGE_TOKEN': True,
+        'LOCALE_FUNC': lambda request: 'kr_KR',
+        'VERIFIED_EMAIL': False,
+        'VERSION': 'v2.4'}}
+
+#facebook
+SOCIAL_AUTH_FACEBOOK_KEY = 'secret!'  # App ID
+SOCIAL_AUTH_FACEBOOK_SECRET ='secret!' #app key
```
Comment (thaodtd-0665) [inline imusic/settings.py]: config biến môi trường vào nhé em

**Comment by khanhlyvu30** [comment]
ready

**Comment by khanhlyvu30** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**